appveyor.yml

```diff
@@ -62,3 +62,5 @@ for:
   test_script:
   - ps: >
       .\run-tests.ps1
+      
+      .\run-tests.ps1 -NodeVersion "8.11.1"

```

Comment:
This is triggering a full test run for all test projects and categories, which is unnecessary and has significant impact on build times. Can we scope this down to re-run only the relevant tests?

Reply:
Yes! I was also thinking of only running on nightly builds? But let me know what you think

Reply:
Ideally the bulk of the node testing will be here: https://github.com/Azure/azure-functions-nodejs-worker/pull/100

That's why I was thinking adding nightly testing might be good? Just as an extra layer of assurance that doesn't need to block or slow down normal development